source/library/CabalGild/Action/EvaluatePragmas.hs

```diff
@@ -25,19 +25,20 @@ run ::
   m ([Fields.Field [Comment.Comment a]], cs)
 run p (fs, cs) = (,) <$> fields p fs <*> pure cs
 
--- | Discovers modules within the given fields.
+-- | Evaluates pragmas modules within the given fields.
 fields ::
   (MonadWalk.MonadWalk m) =>
   FilePath ->
   [Fields.Field [Comment.Comment a]] ->
   m [Fields.Field [Comment.Comment a]]
 fields = mapM . field
 
--- | Discovers modules within the given field. Or, if the field is a section,
--- discover modules recursively within the fields of the section. If modules
--- are discovered for a field, that fields lines are completely replaced. If
--- anything goes wrong while discovering modules, the original field is
--- returned.
+-- | Evaluates pragmas within the given field. Or, if the field is a section,
+-- evaluates pragmas recursively within the fields of the section.
+--
+-- If modules are discovered for a field, that fields lines are completely
+-- replaced. If anything goes wrong while discovering modules, the original
+-- field is returned.
```

Comment:
This comment is the crux of the PR.

Reply:
No it isn't. See https://github.com/tfausak/cabal-gild/pull/12#discussion_r1501955633.